samples/SampleHost/Program.cs

```diff
@@ -14,11 +14,9 @@
 
 void IsVowel(CurrentPlugin plugin, Span<ExtismVal> inputs, Span<ExtismVal> outputs, nint userData)
 {
-    var bytes = plugin.ReadBytes(inputs[0].v.i32);
+    var c = (char)inputs[0].v.i32;
```

Comment:
is there any need for a length check here?

Reply:
I don't think so, the plugin won't be able to call the host function if they don't import it with the right signature

Reply:
ok cool -- thank you!